packages/form-footer/src/FormFooter.tsx

```diff
@@ -20,97 +22,92 @@ import PrimaryButton, { PrimaryButtonProps } from './PrimaryButton';
-              className={buttonStyle}
-              leftGlyph={_cancelButtonProps.leftGlyph}
-              darkMode={darkMode}
-              data-testid="lg-form_footer-cancel_button"
+              leftGlyph={
+                <ArrowLeftIcon data-testid="lg-form_footer-back_button-icon" />
+              }
+              data-testid="lg-form_footer-back_button"
+              {...backButtonProps}
             >
-              {_cancelButtonProps.children || 'Cancel'}
+              {backButtonProps?.children || 'Back'}
             </Button>
           )}
-          {isComponentType(primaryButton as React.ReactElement, 'Button') ? (
-            React.cloneElement(primaryButton as React.ReactElement, {
-              darkMode: darkMode,
-              ['data-testid']: 'lg-form_footer-primary_button',
-            })
-          ) : (
-            <PrimaryButton
-              darkMode={darkMode}
-              data-testid="lg-form_footer-primary_button"
-              {...(primaryButton as PrimaryButtonProps)}
-            />
-          )}
+          <div className={flexEndContent}>
+            {errorMessage && (
+              <Banner className={bannerStyle} variant="danger">
+                {errorMessage}
+              </Banner>
+            )}
+            {showCancelButton && (
+              <Button
+                data-testid="lg-form_footer-cancel_button"
+                {...cancelButtonProps}
+                variant="default"
+              >
+                {cancelButtonProps?.children || 'Cancel'}
+              </Button>
+            )}
+            {showDeprecatedPrimaryButton ? (
+              isComponentType(primaryButton as React.ReactElement, 'Button') ? (
+                React.cloneElement(primaryButton as React.ReactElement, {
+                  darkMode: darkMode,
+                  ['data-testid']: 'lg-form_footer-primary_button',
+                })
+              ) : (
+                <PrimaryButton
+                  darkMode={darkMode}
+                  data-testid="lg-form_footer-primary_button"
+                  {...(primaryButton as PrimaryButtonProps)}
+                />
+              )
+            ) : (
+              <Button
+                variant="primary"
+                data-testid="lg-form_footer-primary_button"
+                {...primaryButtonProps}
+              />
+            )}
+          </div>
         </div>
-      </div>
-    </footer>
+      </footer>
+    </LeafyGreenProvider>
   );
 }
+
+FormFooter.displayName = 'ConfirmationModal';
+
+FormFooter.propTypes = {
+  contentClassName: PropTypes.string,
+  errorMessage: PropTypes.string,
+  darkMode: PropTypes.bool,
+  // eslint-disable-next-line react/forbid-prop-types
+  primaryButtonProps: PropTypes.object,
+  // eslint-disable-next-line react/forbid-prop-types
+  cancelButtonProps: PropTypes.object,
+  // eslint-disable-next-line react/forbid-prop-types
+  backButtonProps: PropTypes.object,
```

Comment:
do we need these eslint disables? I'm not seeing it get flagged in editor. were they failing in CI?

Reply:
![Screenshot 2024-05-21 at 12 42 25 PM](https://github.com/mongodb/leafygreen-ui/assets/5740159/7bfe5e47-7cf2-4b98-930c-2203b92204e0)
 I get a warning without them

Reply:
PropTypes doesn't play nice with more complex TS types